packages/airnode-adapter/src/response-processing/encoding.test.ts

```diff
@@ -161,8 +139,12 @@ describe('Encoding array values', () => {
   });
 
   it('throws on invalid encoding type', () => {
-    expect(() => encodeValue('', 'unit256')).toThrow(`Unknown type: unit256`);
-    expect(() => encodeValue('', 'int256[[]]')).toThrow(`Unknown type: int256[[]]`);
-    expect(() => encodeValue('', '[int256]')).toThrow(`Unknown type: [int256]`);
+    expect(() => encodeValue('', 'unit256')).toThrow(`invalid type (argument=\"type\", value=\"unit256\"`);
+    expect(() => encodeValue('', 'int256[[]]')).toThrow(
+      `unexpected character at position 7 (argument=\"param\", value=\"int256[[]]\"`
```

Comment:
I have reverted back to letting ethers throw the errors. It's not as descriptive as before, but we don't have to check types validity anymore while encoding.

LMK if you'd prefer the previous approach.

Reply:
I'm fine with this

Reply:
Actually, I've changed my mind - I think it will be better to handle it ourselves. It's a bit more code, but the ethers error is not very descriptive and it might be a pain to debug that when looking at AWS logs.